src/_scss/sm-colors-custom-properties.scss

```diff
@@ -40,13 +40,13 @@ html,
   }
 }
 
-@each $id in 1, 2, 3, _success, _error, _warning, _info {
-
-  .sm-palette-#{ $id } {
-    @include apply-palette( $id );
-  }
-
-  .sm-palette-#{ $id }.sm-palette--shifted {
-    @include apply-palette( $id, true );
-  }
-}
+//@each $id in 1, 2, 3, _success, _error, _warning, _info {
+//
```

Comment:
Considering the fact that we generate CSS using `getCSSFromPalettes()` we can safely remove this.